src/lib/tls/tls_session_key.cpp

```diff
@@ -61,15 +61,15 @@ Session_Keys::Session_Keys(const Handshake_State* state,
          salt += state->server_hello()->random();
          }
 
-      m_master_sec = prf->derive_key(48, pre_master_secret, salt);
+      m_master_sec = prf->derive_key(48, pre_master_secret, salt, secure_vector<byte>());
```

Comment:
The master secret expansion constant (EXT_MASTER_SECRET_MAGIC or MASTER_SECRET_MAGIC) could be passed to the TLS PRF here, instead of an empty label.